dataset_manager/data_source.py

```diff
@@ -5,7 +5,11 @@
 module to prepare the datasets
 """
 import os
-import requests
+try:
+    import urllib2
+except Exception:
```

Comment:
Is there a more specific Exception to catch here?

Reply:
Python 3.3-4 and >= 3.5 raise different exception for import error

Reply:
I think there's another way to validate python 2 or 3 for import... I'll check